src/components/player-info.js

```diff
@@ -77,10 +77,10 @@ AFRAME.registerComponent("player-info", {
       let isSkinnedAvatar = false;
       modelEl.object3D.traverse(function(o) {
         if (o.isSkinnedMesh) {
-          const headlessMesh = createHeadlessModelForSkinnedMesh(o);
-          if (headlessMesh) {
+          const mesh = createModelForSkinnedMesh(o);
+          if (mesh) {
             isSkinnedAvatar = true;
-            o.parent.add(headlessMesh);
+            o.parent.add(mesh);
```

Comment:
We(me and @jeus0630) need help this lines.

Default status is not shown head of avatar.
But, we don't know how to toggle avatar head.

So, we change here to avatar head always shows.
If someone helps 'how to toggle avatar head' we can fix here.

Reply:
I think you can do that with the camera layers. Have a look at: https://github.com/mozilla/hubs/blob/cb20d48f4acc7e810928a9b7b302bdab51589c65/src/systems/camera-system.js#L281-L282

I will probably implement that myself this way. Will tell you here if I have sth. to show.